pep-0724.rst

```diff
@@ -100,6 +125,22 @@ checker to further narrow what a ``TypeGuard`` is indicating when it returns
         else:
             # Literal["NW"]
 
+
+This logic can be summed up in a table:
+
+============ ======================= ===================
+|space|      Non strict TypeGuard    Strict TypeGuard
+============ ======================= ===================
+Applies when R not consistent with P R consistent with P
+NP is ..     R                       A & R
+NN is ..     A                       A & ~R
```

Comment:
Should we put these in literal markup (or maths markup?)

Reply:
* https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#math
* https://www.docutils.org/docs/ref/rst/roles.html#math
* https://www.docutils.org/docs/ref/rst/directives.html#math

``:math`...` `` should work, ``.. math:: ...`` for display maths.

Reply:
Thanks for the help. Not sure if that looks better but it might be more consistent with other PEPs that talk about set logic.